server/controllers/email-report/weeklySummaryReport.js

```diff
@@ -0,0 +1,213 @@
+ * @requires NotFound
+ */
+
+
+const uuid = require('node-uuid');
+
+const db = require('../../lib/db');
+const Topic = require('../../lib/topic');
+
+const BadRequest = require('../../lib/errors/BadRequest');
+const NotFound = require('../../lib/errors/NotFound');
+const Period = require('../../lib/period');
+
+const ReportManager = require('../../lib/ReportManager');
+const _ = require('lodash');
+
+const TEMPLATE = './server/controllers/email-report/weeklySummaryReport.handlebars';
+
+exports.view1 = view1;
+exports.weeklySummaryReport = weeklySummaryReport;
+let documentReport;
+
+
+//income by service
+
+function loadServicesIncome() {
+
+  var sql = `SELECT  
+                  SUM(invoice.cost) as sumCost, 
+                  service.name as serviceName, 
+                  CONCAT_WS('.', 'IV', project.abbr, invoice.reference) AS reference,
+                  invoice.reversed
+            FROM invoice
+                JOIN debtor AS d ON invoice.debtor_uuid = d.uuid
+                JOIN service ON service.id = invoice.service_id 
+                JOIN project ON project.id = invoice.project_id
+            WHERE DATE(invoice.date) >= DATE(?) AND 
+                  DATE(invoice.date) <= DATE(?)  
+            GROUP BY service.id `;
+
+
+  var period = new Period(new Date());
+  var week = period.periods.week.limit;
+
+  var monday = '2016-01-01 00:00:00.000';// week.start();
+  var sunday = '2016-12-31 23:59:59.999';// week.end();
+
+  var values = [monday, sunday];
+
+
+  return db.exec(sql, values);
+
+}
+
+function loadPatientData() {
+
+  var period = new Period(new Date());
+
+
+  var week = period.periods.week.limit;
+
+  var monday = week.start();
+  var sunday = week.end();
+
+
+  var Lastweek = period.periods.lastWeek.limit;
+
+  var lastmonday = Lastweek.start();
+  var lastsunday = Lastweek.end();
+
+
+  //patients registered this week
+
+  var sql = `
+          SELECT t1.NumberOfpatientsThisWeek,t2.NumberOfpatientsLastWeek
+           FROM 
+          (
+            SELECT count(uuid) as NumberOfpatientsThisWeek
+            FROM patient
+            WHERE (registration_date BETWEEN ? AND ? )
+          ) as t1,
+          (
+            SELECT count(uuid) as NumberOfpatientsLastWeek
+            FROM patient
+            WHERE (registration_date BETWEEN ? AND ? )
+          ) as t2
+          `;
+
+  var values = [monday, sunday, lastmonday, lastsunday];
+
+  var thisWeekPatients = [];
+
+  return db.exec(sql, values);
+
+}
+
+
+//first and last invoice per day for a week
+function MaxMinInvoiceDate() {
+
+  var sql = `SELECT  MIN(i.date) as minDate, MAX(i.date) as maxDate
+              From invoice i
+              WHERE DATE(i.date) BETWEEN ? AND ?
+              GROUP BY DATE(i.date)`;
+
+  var period = new Period(new Date());
+  var week = period.periods.week.limit;
+
+  var monday = week.start();
+  var sunday = week.end();
+
+  return db.exec(sql, [monday, sunday]);
+}
+
+function view1(req, res, next) {
```

Comment:
This function name has no meaning - it would be better to give this function a meaningful name and maybe a comment describing what the purpose of this function is.